tests/e2e/utils/common.go

```diff
@@ -49,14 +49,13 @@ func GetDeployments(list *metav1.DeploymentList, getDeploymentApi string) error
 
 }
 
-//fucntion to generate deployment body
-func GenerateDeploymentBody(deploymentName, image string, port, replica int) (error, map[string]interface{}) {
+//GenerateDeploymentBody to generate deployment body
+func GenerateDeploymentBody(deploymentName, image, nodeselector string, replica int) (error, map[string]interface{}) {
```

Comment:
I suggest you write methods to create a new Deployment object and then use it rather than building the json in the below fashion.

See https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/deployment/deployment_controller_test.go#L79 for reference.

Reply:
@rohitsardesai83 , Yes referred and done the change accordingly. thanks for the input  :-)